README.md

```diff
@@ -0,0 +1,57 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> This is just a simple project under the supervision of the Microverse platform that tests how descriptive our repo instruction (pull requests description, readme file), also Linters were installed in the project to check for internal code errors and make the code as clean as possible.
+
+# Project Name
+
+> Portfolio: setup and mobile first
```

Comment:
- [ ] These things are unnecessary the making your readme looking not clean Kindly remove them :+1: